File: FIRE DRILL 1/JAVA/TaskEight.java
public class TaskEight {
	public static void main(String[] args) {

	int total = 0;
	for(int count = 1; count <= 10; count++) {
		if (count % 4 == 0) {
		int sum = 0;
	for(int counter = 1; counter <= 5; counter++) {
	 sum += (int)Math.pow(count, counter);
}
	total += sum;
}
}
	System.out.print(total);

}
}

File: FIRE DRILL 1/JAVA SCRIPT/TaskEight.js
const number = 10; 
	
	let total = 0;
	for (let count = 1; count <= number; count++) {
		if (count % 4 == 0) {
			let sum = 0;
	for(let counter = 1; counter <= 5; counter++){	
		sum += (Math.floor(Math.pow(count, counter)));
}
	total += sum;		
}
}	

	console.log (total);

